### DISCUSSION TITLE
How to set requests per connection

### DISCUSSION THREAD

**Discussion by nkuacac** [fortio/fortio#922] [Q&A] [answered]
for example: how to config 2k connections/s  8k requests/s  and to keep 200k connections active(not closed).

**Comment by ldemailly** [comment, marked as answer]
-c is how to set number of connections and -connection-reuse min:max how to indicate how long to keep connections (default is to keep them the whole run)

I am not sure I understand your example though

**Reply by nkuacac** [reply]
Tks. This is my commands: 
`-qps 1000 -c 188 -n 300000 -jitter -uniform --keepalive=true -connection-reuse 4:4 -resolve 172.16.0.131 https://alb.test-nginx.org?size=1000`

**Reply by nkuacac** [reply]
but there is something confuse.
 if i set one pod run :  `-qps 2000 -c 375`. it takes 4cpu.
if i set two pod each run : `-qps 1000 -c 188` . it takes 0.3cpu each.
